frontend/javascripts/admin/job/job_list_view.js

```diff
@@ -91,7 +91,8 @@ class JobListView extends React.PureComponent<Props, State> {
           Tiff export from {job.layerName || "a"} layer of{" "}
           <Link to={`/datasets/${job.organizationName}/${job.datasetName}/view`}>
             {job.datasetName}
-          </Link>
+          </Link>{" "}
+          (Box {job.boundingBox})
```

Comment:
```suggestion
          (Bounding Box {job.boundingBox})
```

would be more consistent with the rest of the UI